tests/integration/test_charm.py

```diff
@@ -171,3 +174,52 @@ async def test_invalid_jvm_config(ops_test: OpsTest) -> None:
                 "Invalid jvm-config, "
                 "Java heap memory specification exceeds application memory constraint"
             )
+    await asyncio.gather(
+        *(
+            [
+                ops_test.model.applications[app_name].set_config({"jvm-config": ""})
+                for app_name in ALL_APP_NAMES
+            ]
+            + [ops_test.model.wait_for_idle(apps=ALL_APP_NAMES)]
+        )
+    )
+
+
+async def test_ingress(ops_test: OpsTest) -> None:
+    """
+    arrange: deploy Spring Boot applications.
+    act: relate the Spring Boot application charm with ingress integrator charm, and update
+        ingress related charm configuration.
+    assert: Ingress integrator charm should create ingress resources in Kubernetes cluster for the
+        Spring Boot charm accordingly.
+    """
+    assert ops_test.model
+    await ops_test.model.add_relation(APP_NAME, INGRESS_NAME)
+    await ops_test.model.wait_for_idle(status=ops.model.ActiveStatus.name)  # type: ignore
```

Comment:
Maybe we have a fixture for `ops.model.ActiveStatus.name` so that we don't have to put `type: ignore` every time we want to use it?

Reply:
Or maybe make it a variable in the test?

Reply:
I moved it to a global variable, thanks!